src/modules/course/course.controller.ts

```diff
@@ -44,8 +44,8 @@ export class CourseController {
     summary: 'get brief information on cards for all courses initially',
   })
   @ApiOkResponse(responsedoc.getAllCourses)
-  async getBreifAllCourses() {
-    return await this.courseService.getBreifAllCourses();
+  async getBreifAllCourses(@Query('skipNum') skipNum: string) {
```

Comment:
Could you write a sample url for hitting this controller?

Reply:
yeah 
![image](https://user-images.githubusercontent.com/59202075/129476179-4afefc10-885d-4450-b94b-90c2005de2b9.png)